src/server/terrain/biomes.zig

```diff
@@ -658,7 +660,7 @@ pub fn deinit() void {
 	biomes.deinit();
 	caveBiomes.deinit();
 	biomesById.deinit();
-	// TODO? byTypeBiomes.deinit(main.globalAllocator);
```

Comment:
Did you actually look into the TODO here or just blindly removed it?

Reply:
I mean, I assumed that it was for adding a map from index to biome, where should I look for more descriptive explanation?

Reply:
So yeah, I blindly removed it C: